ci/build-all-steps.yml

```diff
@@ -72,3 +72,17 @@ steps:
   - powershell: ./ci/stop-collect-diagnostics.ps1
     displayName: Stop collect diagnostics
     condition: and(always(), eq(variables.collect_diagnostics, 'true'), ne(variables['numTasks'], 0))
+
+  - powershell: |
+      # Build the release branch name
+      $startdate = Get-Date -Date '2020-01-20'
+      $enddate = Get-Date
+      $difference = New-TimeSpan -Start $startdate -End $enddate
+      $sprintsSince165 = ($difference.Days/21) - (($difference.Days/21)%1) # Weirdly, PowerShell modulo is actually a remainder operator. This gets the actual modulo value.
+      $currentSprint = $sprintsSince165 + 165
+      $releaseBranch = "damccormtest/releases/m" + $currentSprint
```

Comment:
we probably don't want to have an automated process use your username as part of the branch name

Reply:
Whoops, meant to update branch names - this was for testing

Reply:
I updated one, I left the one that pushes to AzureDevOps because that creates the PR with my PAT (so created by me technically). Not sure if we can get a service account or something to work around that, that can probably slip to a future PR though IMO

Reply:
I should move this into a variable

Reply:
Nevermind, I already did this. Thanks past me

Reply:
Nevermind, I already did this. Thanks past me